flake.nix

```diff
@@ -153,6 +153,13 @@
               extra.buildInputs = [ prev.libdrm ];
               replace.doCheck = false;
             }
+            {
+              attrName = "wbg";
+              extra.buildInputs = [ prev.pixman ];
+              replaceInput = {
+                wayland-protocols = final.new-wayland-protocols;
```

Comment:
does it specifically need the newer one?

Reply:
I don't know? Probably not, I spent a few minutes observing how things were done and  thought that if the project had dependencies that can be provided by `nixpkgs-wayland` they should.

Although  I have not run any tests.

Reply:
> I don't know. Probably not. I've spent a few minutes observing how things were done and thought that if the project had dependencies that can be provided by `nixpkgs-wayland` they should.
> 
> Although I have not run any tests.

If it builds with the wayland-protocols from nixpkgs then the replacement isn't needed